bokehjs/src/coffee/models/glyphs/quad.coffee

```diff
@@ -41,6 +67,23 @@ class QuadView extends Glyph.View
     result['1d'].indices = hits
     return result
 
+  get_anchor_point: (anchor, i, spt) ->
+    left = Math.min(@sleft[i], @sright[i])
+    right = Math.max(@sright[i], @sleft[i])
+    top = Math.min(@stop[i], @sbottom[i])     # screen coordinates !!!
```

Comment:
I understand the idea, which is fine. The problem is that there isn't enough tooling around those concepts. We have methods for converting between those two types of coordinate systems, but that's not enough abstraction to not be bothered with low-level details day-to-day development.